client/cli/src/params/network_configuration_params.rs

```diff
@@ -117,13 +117,7 @@ impl NetworkConfigurationParams {
 			config.network.non_reserved_mode = NonReservedPeerMode::Deny;
 		}
 
-		config.network.sentry_nodes.extend(self.sentry_nodes.clone());
```

Comment:
@tomaka is it intended that you do not set NetworkConfiguration.sentry_nodes anymore??

Reply:
Oh I see, you removed sentry_nodes from NetworkConfiguration. But then why is it still in the cli??

Reply:
Nevermind I will figure out

Reply:
https://github.com/paritytech/substrate/pull/5492/files#diff-40242bf9441f8df803dea44d03cb78d4R115

Moved here.

Reply:
Yeah I figured out. I think sentry_nodes should have been moved to runcmd. NetworkConfigurationParams is only supposed to populate the NetworkConfiguration. I've done it while fixing merge conflicts in #5271